portal-web/test/functional/com/liferay/portalweb/macros/headless/portalinstance/HeadlessPortalInstanceAPI.macro

```diff
@@ -1,5 +1,22 @@
 definition {
 
+	@summary = "Get the groupId for a given company"
```

Comment:
This is technically wrong, since there could be many groups(sites) within a given company, can you revise the summary and macro name? This is retrieving the default group from a company (the one that is first created when you created a new virtual instance)